frontend/src/app/admin/room/room.component.ts

```
@@ -0,0 +1,143 @@
+        console.log(value)
+        this.rooms = value
+      }
+    }
+    )
+  }
+  onRowEditInit(room: any) {
+    this.clonedProducts[room.id as string] = { ...room };
+  }
+
+  onRowEditSave(room: any) {
+    var _ = this;
```

Biome (1.9.4)

[error] This aliasing of this is unnecessary.

Arrow functions inherits `this` from their enclosing scope.
Safe fix: Use this instead of an alias.


(lint/complexity/noUselessThisAlias)